opentelemetry-appender-tracing/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 ### Added
 
 - Add log appender versions to loggers (#1182)
+- Normalized metadata fields with events from the log crate [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
+- Remove rendudant metadata [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
+- Follows [](https://opentelemetry.io/docs/specs/semconv/general/logs/) for metadata names [#1380](https://github.com/open-telemetry/opentelemetry-rust/pull/1380)
```

Comment:
do we follow anything from the linked sem convention anymore?

Reply:
No really. I can remove it.

Reply:
Done.